starter_code/elevator.js

```diff
@@ -1,19 +1,86 @@
+
 class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.direction = "UP";
+    this.passengers = [];
+    this.waitingList = [];
+
+ //   let arr = [...this.passengers, ...this.waitingList];
+
   }
 
-  start() { }
-  stop() { }
-  update() { }
-  _passengersEnter() { }
+  start() { 
+    var upd = setInterval(this.update.bind(this),2000);
```

Comment:
Bien usado el .bind(this). También puedes usar una arrow function de ES6 para que no sea necesario el bindeo:
setInterval(()=>this.update(),1000)

Ten en cuenta que el enunciado dice que se llame a update cada segundo y has puesto 2000ms.